### ISSUE TITLE
Invalid configuration value: failovermethod=priority in /var/lib/qubes/dom0-..(fedora 33 temp)

### ISSUE THREAD

**Issue by TNTBOMBOM** [QubesOS/qubes-issues#6581]
**Qubes OS version**
Qubes R4.0

**Affected component(s) or functionality**
qubes-dom0-update

**Brief summary**
performing qubes update in dom0 will show Invalid configuration

**How Reproducible**
install fedora 33 template , shift to your qubes/dom0 to fedora 33 , remove fedora 32 , perform sudo qubes-dom0-update

**Expected behavior**
perform update without errors 

**Actual behavior**
show errors

**Screenshots**
N/A

**Log**

```
[user@dom0 ~]$ sudo qubes-dom0-update
Using sys-firewall as UpdateVM to download updates for Dom0; this may take some time...
warning: Converting database from bdb to sqlite backend
Invalid configuration value: failovermethod=priority in /var/lib/qubes/dom0-updates/etc/yum.repos.d/fedora.repo; Configuration: OptionBinding with id "failovermethod" does not exist
Invalid configuration value: failovermethod=priority in /var/lib/qubes/dom0-updates/etc/yum.repos.d/fedora.repo; Configuration: OptionBinding with id "failovermethod" does not exist
Invalid configuration value: failovermethod=priority in /var/lib/qubes/dom0-updates/etc/yum.repos.d/fedora.repo; Configuration: OptionBinding with id "failovermethod" does not exist
Invalid configuration value: failovermethod=priority in /var/lib/qubes/dom0-updates/etc/yum.repos.d/fedora-updates.repo; Configuration: OptionBinding with id "failovermethod" does not exist
Invalid configuration value: failovermethod=priority in /var/lib/qubes/dom0-updates/etc/yum.repos.d/fedora-updates.repo; Configuration: OptionBinding with id "failovermethod" does not exist
Invalid configuration value: failovermethod=priority in /var/lib/qubes/dom0-updates/etc/yum.repos.d/fedora-updates.repo; Configuration: OptionBinding with id "failovermethod" does not exist
Fedora 25 - x86_64                              2.5 MB/s |  50 MB     00:19    
Fedora 25 - x86_64 - Updates                    2.5 MB/s |  24 MB     00:09    
Qubes Dom0 Repository (updates)                 327 kB/s | 2.0 MB     00:06    
determining the fastest mirror (14 hosts).. done.--  B/s |   0  B     --:-- ETA
Qubes Templates repository                      1.4 kB/s | 5.9 kB     00:04    
Last metadata expiration check: 0:00:01 ago on Tue May  4 14:17:56 2021.
Dependencies resolved.
Nothing to do.
Complete!
No packages downloaded
Qubes OS Repository for Dom0                     13 MB/s |  26 kB     00:00    
[user@dom0 ~]$ 

```

**Comment by DemiMarie** [comment]
This is harmless.  The fix is simply to delete those lines from the configuration in dom0.

**Comment by andrewdavidwong** [comment]
> This is harmless. The fix is simply to delete those lines from the configuration in dom0.

Another user has [asked](https://www.reddit.com/r/qubesos/comments/n5anuc/strange_error_message_when_getting_updates_for/) about this.

Since this affects all users and will undoubtedly prompt many users to wonder about it, should we push a fix that removes the superfluous lines?

**Comment by DemiMarie** [comment]
> > This is harmless. The fix is simply to delete those lines from the configuration in dom0.
> 
> Another user has [asked](https://www.reddit.com/r/qubesos/comments/n5anuc/strange_error_message_when_getting_updates_for/) about this.
> 
> Since this affects all users and will undoubtedly prompt many users to wonder about it, should we push a fix that removes the superfluous lines?

Yes, we should.  Should I add it to the relevant post-install scripts?

**Comment by andrewdavidwong** [comment]
> > > This is harmless. The fix is simply to delete those lines from the configuration in dom0.
> 
> > 
> 
> > Another user has [asked](https://www.reddit.com/r/qubesos/comments/n5anuc/strange_error_message_when_getting_updates_for/) about this.
> 
> > 
> 
> > Since this affects all users and will undoubtedly prompt many users to wonder about it, should we push a fix that removes the superfluous lines?
> 
> 
> 
> Yes, we should.  Should I add it to the relevant post-install scripts?

I think that's a question for @marmarek.

**Comment by mattmccutchen** [comment]
For the record: It appears that the fix to this issue in R4.0 is in `qubes-release-4.0-10`, which is now in the `qubes-dom0-current` repository.  It seems strange to me that `qubes-release-4.0-10` is not mentioned in the thread above; there may be some aspect of the update process that I don't understand.

I was still experiencing this issue yesterday, and since I didn't see a `qubes-release` update for R4.0 above, I thought this issue might have been closed without updating R4.0 even though it has the "Release 4.0 updates" milestone, which [would be a mistake](https://github.com/QubesOS/qubes-issues/issues/6321#issuecomment-850905225).  But in fact, it appears that something was wrong with `qubes-dom0-update` on my system that prevented it from installing `qubes-release-4.0-10` (possibly related to #6321), and somehow that problem has now gone away and `qubes-release-4.0-10` got installed and this issue is now fixed on my system.

So if anyone else is still experiencing this issue on R4.0, check if you have `qubes-release-4.0-10` installed, and if not, fiddle with `qubes-dom0-update` (and apply a workaround for #6321 if needed) until it offers to install `qubes-release-4.0-10`.
